magento/pwa-studio
# [PWA-1029] HomePage unstyled in scaffolded app

## Description

By default, when building an app, the edition of the Magento Store is set to `CE`.
The custom CSS for the Home Page is only called in the `EE` version.

With this update, the default will now be `EE` and we will also ask the question when creating the app.

## Related Issue

Closes https://jira.corp.magento.com/browse/PWA-1029.

## Acceptance

### Verification Stakeholders

## Verification Steps

#### Test scenario(s) for direct fix/feature

- Test scaffolding. Remove or edit the `--backend-edition` parameter. The .env file should reflect your changes.
- Run `packages/pwa-buildpack/bin/buildpack create-project scaffold-test --template "@magento/venia-concept" --name "scaffold-test" --author "test123 <tester@example.com>" --backend-url "https://master-7rqtwti-c5v7sxvquxwl4.eu-4.magentosite.cloud/" --braintree-token "sandbox_8yrzsvtm_s2bg8fs563crhqzk" --npm-client "yarn" --install false --backend-edition="EE"`.
- Test project creation. You should have a prompt asking you for the Magento Edition. The .env file should reflect your choices.
- Run `packages/create-pwa/bin/create-pwa`.

#### Test scenario(s) for any existing impacted features/areas

- Run `yarn test`.
- Run `cd packages/create-pwa/ && yarn jest`.
